libs/core/radio/radio-button/radio-button.component.ts

```diff
@@ -97,6 +97,13 @@ export class RadioButtonComponent<T = any>
     @Input()
     disabled = false;
 
+    /** The field is used to tell if radio button should be read only
+     * Value: true or false
```

Comment:
The value and default comments are redundant as they're automatically inferred and generated into the API docs